models/model_upload/llms/lmdeploy-llama-3_2-1b-instruct/requirements.txt

```diff
@@ -0,0 +1,15 @@
+torch==2.4
+tokenizers
+transformers
```

Comment:
We should also add dependencies versions with the package name for each of depencencies

Reply:
Added version specifications for key packages: transformers, protobuf, torch, etc. The remaining packages will be managed by pip to align with the specified versions.